html/statbrowser.js

```diff
@@ -42,20 +42,152 @@ var turfcontents = [];
 var turfname = "";
 var imageRetryDelay = 500;
 var imageRetryLimit = 50;
+var topButtons = document.getElementById("top-buttons");
 var menu = document.getElementById("menu");
+var endMenu = document.getElementById("end-menu")
 var under_menu = document.getElementById("under_menu");
 var statcontentdiv = document.getElementById("statcontent");
 var storedimages = [];
 var split_admin_tabs = false;
 
+let toLookUp = [
+	"icon-size",
+	"text-mode",
+	"zoom-mode"
+]
+var lookedUpProperties = {}
+
+let bigButtons = [
+	{
+		name: "Changelog",
+		command: "Changelog",
+		class: "changelog",
+		icon: "list-ul"
+	},
+	{
+		name: "Rules",
+		command: "rules",
+		icon: "scale-balanced"
+	},
+	{
+		name: "Wiki",
+		command: "wiki",
+		icon: "book"
+	},
+	{
+		name: "Forum",
+		command: "forum",
+		icon: "envelope"
+	},
+	{
+		name: "Submit Bug",
+		command: "submit-bug",
+		class: "bug-button",
+		icon: "bug"
+	},
+	{
+		name: "Discord",
+		command: "discord",
+		class: "discord-button",
+		icon: "comments"
+	}
+]
+
+let clientButtons = {
+	"File": [
+		{name: "Take Screenshot	(F2)", command: ".screenshot auto"},
```

Comment:
```suggestion
		{name: "Take Screenshot	(F2)", command: ".screenshot auto"},
```
This doesn't even work as far as I can tell, probably better to remove it or have it print out where it saves the picture

Reply:
it does, and because it's just clientside we can't add any more detail to it